conan/tools/apple/xcodebuild.py

```diff
@@ -26,18 +28,29 @@ def _sdkroot(self):
             sdk = "{}{}".format(self._sdk, self._sdk_version)
         return "SDKROOT={}".format(sdk) if sdk else ""
 
-    def build(self, xcodeproj, target=None):
+    def build(self, xcodeproj, target=None, configuration=None, build_options=[]):
```

Comment:
Passing a mutable object as default arg in Python is bad practice. It is better to pass ``build_options=None``, then do ``build_options = build_options or []`` in code.  

See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments